### DISCUSSION TITLE
finalizing 的翻譯

### DISCUSSION THREAD

**Discussion by robinsonweng** [python/python-docs-zh-tw#762] [Add Term]
### 初步確認

- [X] 我已經確認過 [術語列表](https://github.com/python/python-docs-zh-tw/wiki/%E8%A1%93%E8%AA%9E%E5%88%97%E8%A1%A8) 中沒有相關術語。
- [X] 我已經確認過 [Python 官方術語列表](https://docs.python.org/zh-tw/3/glossary.html) 中沒有相關術語。

### 術語原文

finalizing

### 術語翻譯

終結, 完成

### 說明

finalizing 一詞有的地方翻譯為終結, 有的地方翻譯為完成項, 不確定要怎翻譯, 目前暫定翻譯為終結
[出處](https://github.com/python/python-docs-zh-tw/blob/3.12/library/asyncio-runner.po#L52)

### 參考資料

https://openhome.cc/Gossip/JavaEssence/Finalize.html
https://learn.microsoft.com/zh-tw/dotnet/csharp/programming-guide/classes-and-structs/finalizers

**Comment by ken71301** [comment]
我個人以語意上來說會比較想翻成 `最終化`（from 樂詞網）
劍橋詞典的解釋：**to make a final and certain decision about a plan, date, etc.**
但舊的翻譯上的確翻成了終結，困擾。

**Reply by ken71301** [reply]
3/30：參考目前文件內所有finalize / finalizing 用法後，其意義應該為`定案`或`最終化`某個物件，確定不該翻譯為`終結`
討論結果：finalize 與 finalizing 皆翻譯為 `最終化`
